components/omega/doc/design/ShallowWaterOmega0.md

```diff
@@ -0,0 +1,212 @@
+This equation set does not include any vertical advection or diffusion. Omega-0 will have a vertical index for performance testing and future expansion, but vertical layers will be simply redundant.
+
+To do: Complete table of symbol definitions. See [markdown table generator](https://www.tablesgenerator.com/markdown_tables)
+| symbol | name      | units  |
+|--------|-----------|--------|
+| $u$    | velocity  | m/s    |
+| $h$    | thickness | m      |
+| $\phi$ | tracer    | varies |
+| $t$    | time      | s      |
+
+### 2.2 Requirement: Numerical method will be the TRiSK formulation
+
+The horizontal discretization will be taken from 
+[Thuburn et al. 2009](https://www.sciencedirect.com/science/article/pii/S0021999109004434) and [Ringler et al. 2010](https://www.sciencedirect.com/science/article/pii/S0021999109006780), as described in detail in the algorithmic formulation in Section 3 below. This is the same base formulation as in MPAS-Ocean. In addition, we will consider small alterations from the original MPAS-Ocean horizontal discretization and include them as options if they are of minimal additional effort. This includes the recent AUST formulation in [Calandrini et al. 2021](https://www.sciencedirect.com/science/article/pii/S146350032100161X) and simple vorticity averaging considered by the Omega team this past year.
+
+### 2.3 Requirement: Omega-0 will use MPAS format unstructured-mesh domains
+
+We will continue to use the MPAS-format netcdf files for input and output, with the same mesh variable names and dimensions. This will facilitate ease of use and interoperability between MPAS-Ocean and Omega.
+
+Omega mesh information will be stored in a separate mesh file, and not be included with initial condition, restart, or output files. This way there is never redundant mesh data stored in these files.
+
+To do: add list of minimum number of variables that defines the mesh, and the additional variables that can be computed on start-up.
+
+### 2.4 Requirement: Omega-0 will interface with polaris for preprocessing and postprocessing
+
+A substantial investment has already been made in the polaris tools. Continuing the use of polaris for Omega will speed development and encourage the documentation and long-term reproducibility of test cases.
+
+The test cases relevant to this design document are in Section 5 below.
+
+### 2.5 Requirement: Omega-0 will run portably on various DOE architectures (CPU and GPU nodes)
+
+Omega will be able to run on all the upcoming DOE architectures and make good use of GPU hardware. This should occur with minimal alterations in the high-level PDE solver code for different platforms. 
+
+The proposed solution is to use YAKL [(Norman et al. 2023)](https://link.springer.com/10.1007/s10766-022-00739-0) for within-node, shared memory computations. All large arrays will be YAKL objects, and computations that loop over array indices will use YAKL function calls.
```

Comment:
Pedantic but maybe this should be in Design, no in requirements?

Reply:
Good point. I changed this to a list of possible options to provide some background, but will not specify the solution until the design section.